### PR TITLE
added new test cases in endpoints

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
https://spryker.atlassian.net/browse/CC-16767
https://spryker.atlassian.net/browse/CC-16765
https://spryker.atlassian.net/browse/CC-16772
 added new test-case

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot [hunk 1]
```diff
@@ -46,4 +46,30 @@ Get_agent_token_with_wrong_type
...
\ No newline at end of file
+    And Response should return error message:    Invalid type.
+
+Get_agent_access_token_by_empty_email_and_empty_password
+    When I send a POST request:    /agent-access-tokens    {"data": {"type": "agent-access-tokens","attributes": {"username": "","password": ""}}}
+    Then Response status code should be:    401
+    And Response reason should be:    Unauthorized
+    And Response should return error code:    4101
+    And Response should return error message:    Failed to authenticate an agent.
+Get_agent_access_token_with_blank_spaces
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot:57]: I do not think this test is very valuable. Please change this test to have this body
{"data": {"type": "agent-access-tokens","attributes": {}}}

File: tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot [hunk 2]
```diff
@@ -46,4 +46,30 @@ Get_agent_token_with_wrong_type
...
+    When I send a POST request:    /agent-access-tokens    {"data": {"type": "agent-access-tokens","attributes": {"username":" " ,"password":" "}}}
+    Then Response status code should be:    401
+    And Response reason should be:    Unauthorized
+    And Response should return error code:    4101
+    And Response should return error message:    Failed to authenticate an agent.
+Get_agent_access_token_by_non_agent_email_and_password
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot:63]: we already have such test above called Get_agent_token_for_user_who_is_not_agent

File: tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot [hunk 3]
```diff
@@ -46,4 +46,30 @@ Get_agent_token_with_wrong_type
...
+    When I send a POST request:    /agent-access-tokens    {"data": {"type": "agent-access-tokens","attributes": {"username": "${yves_user.email}","password": "${yves_user.password}"}}}
+    Then Response status code should be:    401
+    And Response reason should be:    Unauthorized
+    And Response should return error code:    4101
+    And Response should return error message:    Failed to authenticate an agent.
+
+Get_agent_access_token_by_invalid_email_and_invalid_password
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot:70]: you cannot have an invalid password for a user that does not exist.
Please rename to 
Get_agent_access_token_by_non_existent_user

File: tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot (hunk 1 above)
Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot:57]: Removed

File: tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot (hunk 2 above)
Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot]: Removed

File: tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot (hunk 3 above)
Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/agent_endpoints/agent_access_tokens/negative.robot]: Renamed

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**